app/views/root/400.html.erb

```diff
@@ -1,6 +1,5 @@
 <%= render partial: "error_page", locals: {
-    title: "Bad request",
-    heading: "Bad request",
-    intro: '',
+    heading: "Sorry, there is a problem",
+    intro: '<p>Go back and change the information you entered.</p>',
```

Comment:
Can you add a class of `govuk-body` to this and the other `<p>` tags passed to this partial? It probably looks fine now but we're going to be removing a lot of the default styling in static fairly soon.

Reply:
That's a good point. Considering there are a few other items on this page - such as the title - I'm worried this could lead to scope creep. Since it sounds like we're going to audit the whole app anyway (to remove the default styling), could we leave it until then?

Reply:
That's fine, as long as it gets looked at at some point.